### PR TITLE
feat:  add seasonal learn alert

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX
This pr adds a seasonal learn CTA that links to /donate.
It will show a different button variation based on the user's email address for A/B testing.
Screen Shot 2021-11-30 at 5 11 42 PM

### CODE REVIEW

File: client/src/components/Intro/index.tsx [hunk 1]
```diff
@@ -24,9 +28,16 @@ const Intro = ({
...
   completedChallengeCount,
-  slug
+  slug,
+  email,
+  onAlertClick
 }: IntroProps): JSX.Element => {
   const { t } = useTranslation();
+  const buttonVariation = (email: string | undefined): string => {
+    if (!email || emailToABVariant(email).isAVariant)
+      return t('buttons.donate');
```
Comment (ojeytonwilliams) [inline client/src/components/Intro/index.tsx:38]: Does it matter that we will get more As than Bs?

File: client/src/pages/learn.tsx [hunk 2]
```diff
@@ -67,6 +76,17 @@ function LearnPage({
 }: LearnPageProps) {
   const { t } = useTranslation();
 
+  const onAlertClick = () => {
+    console.log('click');
```
Comment (ojeytonwilliams) [inline client/src/pages/learn.tsx:80]:
```suggestion
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: client/src/components/Intro/index.tsx (hunk 1 above)
Comment (ahmadabdolsaheb) [reply client/src/components/Intro/index.tsx:38]: We should not and it does not matter. That block is only shown when user is signed. Also, If there are no emails it means the user is not signed it. We only report signed in users in GA saga.

Comment (ojeytonwilliams) [reply client/src/components/Intro/index.tsx:38]: Great, thanks for explaining.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by nhcarrigan** [comment]
image

Does it stand out enough on the light theme?

**Comment by ahmadabdolsaheb** [comment]
> Does it stand out enough on the light theme?
I see your point. 

Since alerts in general have the same colors regardless of our theme, they would have more contrast in one of the themes. 

That is something that we could address in the new ui comp library. 

For now, it is just an info alert component with a cheerful holiday background :)

**Review by naomi-lgbt — APPROVED** [review]
I'm happy with this as-is, then. 🙂

**Pull request merged**